static/app/views/starfish/queries/useReleases.tsx

```diff
@@ -10,7 +10,7 @@ import {useTableQuery} from 'sentry/views/starfish/views/screens/screensTable';
 
 export function useReleases(searchTerm?: string) {
   const organization = useOrganization();
-  const {selection} = usePageFilters();
+  const {selection, isReady} = usePageFilters();
```

Comment:
Small nit, in one file you just call it `isReady`, but the other you call `isPageFiltersReady`

```suggestion
  const {selection, isReady: isPageFiltersReady} = usePageFilters();
```